### DISCUSSION TITLE
What is the vision+mission of FreshRSS?

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#5322] [General]
Hi,
let's try to describe the idea/vision/mission behind FreshRSS. What is important to know about this application? What drives us forward? What is important for us/the application?

I know, that @Alkarex has some very nice points. I can remember:
- be a friendly community (f.e. respect each other)
- be sustainable to the environment:
  - support also older technologies, so that older servers could be used (f.e. support older PHP versions https://github.com/FreshRSS/FreshRSS/discussions/3321#discussioncomment-835704 )
  - do not consume to much resources from other servers (f.e. do not pull feeds to often)
- Support the Open Source + RSS community
- keep the technology simple, so that it is easy to install and maintain, reduce external dependencies (no external PHP framework, no front end JavaScript framework etc.)

What else?

**Comment by Alkarex** [comment]
You got the main points 👍🏻 
In point 2, not too much resources from our own server as well, and the aspect about supporting older systems can be generalised to avoid contributing to premature obsolescence.
In point 3, we could link to efforts / vision from e.g. Framasoft, YunoHost.
In point 4, stick in particular to standards / vanilla code (e.g. use SQL standard instead of using another layer of whatever). And also to ease self-hosting.
In French, but here is a journalistic article covering some of the topic https://www.nextinpact.com/article/28150/106286-intelligence-social-freshrss-veut-redonner-coup-fouet-aux-agregateurs-flux
